### ISSUE TITLE
Allow users to edit the title of a forum thread

### ISSUE THREAD

**Issue by irilyth** [buttonmen-dev/buttonmen#1560]
People can edit messages that they post, but it'd be nice if they could also edit the titles of the threads that they start. (I just created one called "RandomBM", which I now realized is overly general, and it might be nice to give it a more specific name, but I can't edit it.)

**Comment by nkmcalli** [comment]
Is changing the title the best way?  If you want to change the title, maybe starting a new thread is better?  Changing the title could make it difficult for folks to locate an old thread that they are trying to find.

So for your Random BM example, you can start a new thread and then put a message with a link to the new thread.

What if someone goes looking for Random BM thread and they can't find it?  :)

**Comment by irilyth** [comment]
Yeah, I think you wouldn't generally want to change the title of an old thread, much like you wouldn't want to go back an edit an old post; I think it's mostly useful in cases where you make a mistake in the title and want to fix it soon after. (In my example, http://www.buttonweavers.com/ui/forum.html#!threadId=109, I realized right away that I should have named it "Who likes to play RandomBM games" or something like that.)

**Comment by nkmcalli** [comment]
:)
